src/robusta/runner/config_loader.py

```diff
@@ -231,6 +231,11 @@ def __reload_playbook_packages(self, change_name):
                 self.registry.set_playbooks(playbooks_registry)
                 self.registry.set_sinks(sinks_registry)
 
+                telemetry = self.registry.get_telemetry()
+                telemetry.playbooks_count = len(runner_config.active_playbooks)
+                telemetry.account_id = hashlib.sha256(str(runner_config.global_config.get("account_id")).encode("utf-8")).hexdigest()
```

Comment:
I would use some default value, in case account_id/cluster_name isn't configured in global_config